vector/src/main/java/im/vector/app/features/home/room/detail/composer/RichTextComposerLayout.kt

```diff
@@ -237,14 +237,16 @@ internal class RichTextComposerLayout @JvmOverloads constructor(
         views.composerEditTextOuterBorder.background = borderShapeDrawable
 
         setupRichTextMenu()
-        views.richTextComposerEditText.mentionDisplayHandler = object : MentionDisplayHandler {
-            override fun resolveMentionDisplay(text: String, url: String): TextDisplay =
-                pillDisplayHandler?.resolveMentionDisplay(text, url) ?: TextDisplay.Plain
-
-            override fun resolveAtRoomMentionDisplay(): TextDisplay =
-                pillDisplayHandler?.resolveAtRoomMentionDisplay() ?: TextDisplay.Plain
-        }
-
+        views.richTextComposerEditText.updateStyle(
+                styleConfig = views.richTextComposerEditText.styleConfig,
+                mentionDisplayHandler = object : MentionDisplayHandler {
+                    override fun resolveMentionDisplay(text: String, url: String): TextDisplay =
+                            pillDisplayHandler?.resolveMentionDisplay(text, url) ?: TextDisplay.Plain
+
+                    override fun resolveAtRoomMentionDisplay(): TextDisplay =
+                            pillDisplayHandler?.resolveAtRoomMentionDisplay() ?: TextDisplay.Plain
+                }
+        )
```

Comment:
`mentionDisplayHandler` is now private, so using `updateStyle` with existing `styleConfig` for the first parameter. There is maybe another way to set `mentionDisplayHandler`?

Reply:
This is the right way to do it now. We did this to minimize the number of times we need to re-create the underlying `HtmlToSpanConverter`, specially in Compose.